.github/CODEOWNERS

```diff
@@ -1,2 +0,0 @@
-# Order is important; the last matching pattern takes the most precedence.
-* @raulriera
```

Comment:
Does this mean non-members can merge now too?

Reply:
It just means I won't be added as a reviewer all the time

Reply:
_requests changes_